.github/renovate.json

```diff
@@ -14,12 +14,30 @@
       "enabled": false
     },
     {
```

Comment:
Wanna change the timing of the lock update to be more frequent now or save it for later?

Reply:
Can save it for later. I want to see it make PRs we can just click merge on without headache a couple times before upping it. Might try it on depedency dashboard a couple times.